pkg/auth/creds_config.go

```diff
@@ -0,0 +1,136 @@
+package auth
+
+import (
+	"fmt"
+	"os/exec"
+
+	"github.com/notaryproject/notation/pkg/config"
+	dockerConfig "github.com/notaryproject/notation/pkg/dockerconfig"
+)
+
+type ConfigFile struct {
+	CredentialsStore  string            `json:"credsStore,omitempty"`
+	CredentialHelpers map[string]string `json:"credHelpers,omitempty"`
+}
+
+// loadConfig loads the configuration
+// from the config file
+func loadConfig() (*ConfigFile, error) {
+	config, _ := loadNotationConfig()
```

Comment:
We should always check error and never ignore them.

Reply:
thanks for reminding!